apollo-client/v3/schema-driven-testing/src/__tests__/jest/products.test.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  createSchemaFetch,
+  createTestSchema,
+} from "@apollo/client/testing/experimental";
+import { makeExecutableSchema } from "@graphql-tools/schema";
+import { render, screen } from "@testing-library/react";
+import graphqlSchema from "../../../schema.graphql";
+import { makeClient } from "../../client";
+import { ApolloProvider } from "@apollo/client";
+import { Products } from "../../products";
+
+const staticSchema = makeExecutableSchema({ typeDefs: graphqlSchema });
+
+const schema = createTestSchema(staticSchema, {
+  resolvers: {
+    Query: {
+      products: () =>
+        new Array(5).fill(null).map((_, i) => ({
+          id: i,
+          mediaUrl: `https://example.com/image${i}.jpg`,
+        })),
+    },
+  },
+  scalars: {
+    Int: () => 6,
+    Float: () => 22.1,
+    String: () => "default string",
+  },
+});
+
+describe("Products", () => {
+  it("renders", async () => {
+    const forkedSchema = schema.fork();
```

Comment:
Maybe this is for my own education, but since you aren't making changes to the resolvers, do you need to fork it?

Regardless, this might be good signal for our documentation to write out when you need `.fork` vs being fine without it.

Reply:
Ha, I think you might have removed this after I added this comment, so please ignore.

Reply:
Haha yes - I may add another example with `.fork` here in the future but for now I just removed it :)